soc/arm/common/cortex_m/arm_mpu_regions.c

```diff
@@ -21,13 +24,24 @@ static const struct arm_mpu_region mpu_regions[] = {
 #endif
 	/* Region 1 */
 	MPU_REGION_ENTRY("SRAM_0",
+#if USE_PARTITION_MANAGER
```

Comment:
If doing this, then please ensure users are unable to set the `SRAM_SIZE` and `SRAM_BASE_ADDRESS` when `USE_PARTITION_MANAGER=y`.
https://github.com/nrfconnect/sdk-zephyr/blob/41ce8e165a9af4171ef52a73aede89da6ce60920/arch/Kconfig#L130-L144
by adding:
```
  depends on !USE_PARTITION_MANAGER
```

Reply:
Will try it!

Reply:
Discussed offline, posting the conclusion here for others to see: It won't work since `USE_PARTITION_MANAGER` is not available as a kconfig symbol.